package.json

```diff
@@ -20,7 +20,6 @@
     "axios": "^0.19.2",
     "bootstrap": "^4.4.1",
     "date-fns": "^2.9.0",
-    "dotenv": "^8.2.0",
```

Comment:
Why are we removing dotenv?

Reply:
@debauchery1st

Reply:
Trevor: "react doesn't need it when the .env file is placed in the root and all environment vars are prefixed with REACT_APP_"

Reply:
https://create-react-app.dev/docs/adding-custom-environment-variables/